test/utils/utils.go

```diff
@@ -18,6 +18,24 @@ import (
 	. "github.com/onsi/gomega/gexec" //nolint:golint,stylecheck
 )
 
+type NetworkBackend int
+
+const (
+	// Container Networking backend
+	CNI NetworkBackend = iota
+	// Netavark network backend
+	Netavark NetworkBackend = iota
+)
+
+func (n NetworkBackend) ToString() string {
+	switch n {
+	case CNI:
+		return "cni"
+	default:
+		return "netavark"
```

Comment:
Grrr, github ate my comment.

I find this sort of usage uncomfortable, maintenance-wise. Is there any reason not to do:
```
    case Netavark:
      return "netavark"
   default:
      panic("zomg this cannot happen")
```

Reply:
we cannot panic or the whole suite would panic.  i however did add a logrus and returning an empty string should suffice.

Reply:
Just curious, this is not a request for change: what exactly would be so bad about the entire test suite panicking? I mean, if it ever gets to that place with an unknown network, isn't that precisely the kind of situation where you **want** to panic?